mirobo/plug_v1.py

```diff
@@ -0,0 +1,54 @@
+from .device import Device
+
+
+class PlugV1(Device):
+    """Main class representing the chuangmi plug v1."""
+
+    def on(self):
+        """Power on."""
+        return self.send("set_on", [])
+
+    def off(self):
+        """Power off."""
+        return self.send("set_off", [])
+
+    def usb_on(self):
+        """Power on."""
+        return self.send("set_usb_on", [])
+
+    def usb_off(self):
+        """Power off."""
+        return self.send("set_usb_off", [])
+
+    def status(self):
+        """Retrieve properties."""
+        properties = ['on', 'usb_on']
+        values = self.send(
+            "get_prop",
+            properties
+        )
+        return PlugV1Status(dict(zip(properties, values)))
+
+
+class PlugV1Status:
+    """Container for status reports from the plug."""
+    def __init__(self, data: Dict[str, Any]) -> None:
```

Comment:
An example would also be nice here :-) Are those values already booleans or should they be converted below?